script.js

```diff
@@ -0,0 +1,94 @@
+const categories = [
+  {
+    key: 1,
+    img_src: './assets/souchi.jpg',
+    img_atr: 'Sushi',
+    title: 'Sushi',
+    description: 'necessitatibus tenetur. Ea, numquam cupiditate.',
+    content: 'sit amet consectetur adipisicing iste neque optio earum labore, ipsam',
+  },
+  {
+    key: 2,
+    img_src: './assets/pizza.jpg',
+    img_atr: 'Pizza',
+    title: 'Pizza',
+    description: 'Lorem ipsum dolossumenda maaceat voluptatem asperiores',
+    content: 'repellat iste neque optio earum labore,   necessitatibus. Ea, numquam cupiditate.',
+  },
+  {
+    key: 3,
+    img_src: './assets/salade.jpg',
+    img_atr: 'Salad',
+    title: 'Salad',
+    description: '',
+    content: 'ipsum dolor sit . Pariatur e optio earum labore, ipsam facilis ',
+  },
+  {
+    key: 4,
+    img_src: './assets/ramen.jpg',
+    img_atr: 'Ramen',
+    title: 'Ramen',
+    description: 'Pariatur fugiatneque  voes eos illum necessitatibus tenetur.',
+    content: ' dolor sit amet consectetur adipisbore, ipsam facilis assumenda magni',
+  },
+  {
+    key: 5,
+    img_src: './assets/Fast-Food.jpg',
+    img_atr: 'Fast Food',
+    title: 'Fast Food',
+    description: 'Lorem ipsum dolnsectetur adipisicing elit.giat repellat iste neque optio',
+    content: 'voluptatem asperiores eos illum necesnetur. Ea, numquam cupiditate.',
+  },
+];
```

Comment:
- [ ] Good job dynamically rendering this section of the page and using the right semantic HTML tags to group them. However, there are a few issues you need to address:
- You're expected to have 6 articles, not 5.
- The image should have a decorative shadow at the top left corner.
- You should also endeavour to add some margin/padding to the left and right sides of this section as it is in the design below

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124148726-d3930580-da87-11eb-9155-b66f03fdfdc5.png) | ![image](https://user-images.githubusercontent.com/16058470/124148797-e9a0c600-da87-11eb-9652-ce60f6ff4da1.png)


On mobile view,
- [ ] The articles should be centred with an equal amount of space to the left and right sides.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124150811-d3940500-da89-11eb-8cb3-047849562acc.png) | ![image](https://user-images.githubusercontent.com/16058470/124150906-ea3a5c00-da89-11eb-97e6-f657fdc007d9.png)

Reply:
ok i did it now